src/main/java/com/chrisrm/idea/utils/IconReplacer.java

```diff
@@ -23,11 +26,16 @@ public static void replaceIcons(Class iconsClass, String iconsRootPath) {
                         setFieldValue(value, "myWasComputed", Boolean.FALSE);
                         setFieldValue(value, "myIcon", null);
                     } else if (byClass.getName().endsWith("$CachedImageIcon")) {
-                        patchUrlIfNeeded(value, iconsRootPath);
+                        String p = patchUrlIfNeeded(value, iconsRootPath);
+                        Icon newIcon = IconLoader.getIcon(p);
+						setFinalStatic(field, newIcon);
```

Comment:
A bit nit-picky, but I'd fix the indentation here and give the variable `p` a proper name.